### DISCUSSION TITLE
send data to radiosondy

### DISCUSSION THREAD

**Discussion by DF7RW** [dl9rdz/rdz_ttgo_sonde#411] [TTGO firmware]
Hello, I am sending my RX data to Sondehub so far and would also like to send to radiosondy.info
Is this possible and what settings do I need?

**Comment by DG0MG** [comment]
Ja, natürlich ist das möglich. 
Lies bitte erstmal im Wiki, da ist schon relativ viel erklärt: [https://github.com/dl9rdz/rdz_ttgo_sonde/wiki/Data-feed-configuration#aprs-parameters-for-example-for-radiosondyinfo](url)
Der "APRS-Passcode" ist eine Art Passwort, das kannst Du Dir hier generieren lassen: [https://apps.magicbug.co.uk/passcode/](url)

**Comment by DF7RW** [comment]
Hallo Steffen,Danke für die Info. Funktioniert jetzt73RawanVon meinem iPhone gesendetAm 27.12.2023 um 12:40 schrieb Steffen ***@***.***>:﻿
Ja, natürlich ist das möglich.
Lies bitte erstmal im Wiki, da ist schon relativ viel erklärt: https://github.com/dl9rdz/rdz_ttgo_sonde/wiki/Data-feed-configuration#aprs-parameters-for-example-for-radiosondyinfo
Der "APRS-Passcode" ist eine Art Passwort, das kannst Du Dir hier generieren lassen: https://apps.magicbug.co.uk/passcode/
